### DISCUSSION TITLE
Does apisix support to handle problems nodes?

### DISCUSSION THREAD

**Discussion by ftssyang** [apache/apisix#5887] [Q&A]
![WSEO%SPV4`OHF`H5C(C$K3H](https://user-images.githubusercontent.com/16313949/147088532-61b7277c-dc36-4112-adfc-246f0e13abb7.png)
比如目标节点  8082 有问题 直接屏蔽掉

**Comment by tokers** [comment]
See https://apisix.apache.org/docs/apisix/health-check/.
